collections/card4l.yaml

```diff
@@ -0,0 +1,122 @@
+Name: Sentinel-1 Radiometric Terrain Corrected SAR Backscatter
+Description: |
+ Sentinel-1 Radiometric Terrain Corrected SAR Backscatter is a product processed from [Sentinel-1 ground range detected (GRD)](https://docs.sentinel-hub.com/api/latest/data/sentinel-1-grd/) and is compliant with [CEOS Analysis Ready Data for Land (CARD4L)](https://ceos.org/ard/) specifications for Normalised Radar Backscatter (NRB) products. CARD4L compliant products fulfil requirements for general metadata, per-pixel metadata, radiometric and atmospheric corrections and geometric corrections outlined in the NRB [product family specifications](https://ceos.org/ard/files/PFS/NRB/v5.5/CARD4L-PFS_NRB_v5.5.pdf).
+ These products are processed using Sentinel Hub [CARD4L generation tool](https://apps.sentinel-hub.com/s1-card4l/) and are radiometrically calibrated to γ° (gamma-nought) backscatter, and provided as linear backscatter power. Full radiometric terrain correction (RTC) based on the technique developed by David Small[1], has been applied using the [Copernicus DEM](https://docs.sentinel-hub.com/api/latest/data/dem/) in order to mitigate topographic terrain effects on the backscatter induced by side-looking geometry of SAR imaging and compounded by rugged terrain. Speckle filtering has not been applied in order to preserve spatial resolution and user freedom. 
```

Comment:
Not sure if `[1]` in `David Small[1]` is necessary in this (`yaml`) file because there is no references in this file.

If you wish to still have the reference / link to the article in the description in this file, there are more options:
- you can make a link out of `technique developed by David Small` (so that would appear as a link)
- you can change the sentence to `... based on the technique developed by David Small and described in the article [Flattening Gamma: Radiometric Terrain Correction for SAR Imagery](https://doi.org/10.1109/TGRS.2011.2120616)` 
- something similar